test/lib/mayaUsd/render/mayaToHydra/CMakeLists.txt

```diff
@@ -57,6 +56,11 @@ if(MayaUsd_FOUND)
     list(APPEND INTERACTIVE_TEST_SCRIPT_FILES testMayaUsdAPIUsage.py)
 endif(MayaUsd_FOUND)
 
+# Code coverage is not computed for the Hydra scene browser.
+if (NOT CODE_COVERAGE)
```

Comment:
Scene browser is not built for coverage build:
https://github.com/Autodesk/maya-hydra/blob/ef17d5013adfdaffde3d5d0e3946be69970649b6/CMakeLists.txt#L165